xapian-core/cluster/cluster.cc

```diff
@@ -1,8 +1,8 @@
 /** @file cluster.cc
- *  @brief Xapian::Cluster API class
+ *  @brief Cluster API
  */
-/* Copyright (C) 2016 Richhiey Thomas
- * Copyright (C) 2016 Richard Boulton
+/* Copyright (C) 2010 Richard Boulton
+ * Copyright (C) 2016 Richhiey Thomas
```

Comment:
Yeah, it takes a while to get into the habit of breaking things into distinct commits :-)

(`git add -p` and friends helps.)